content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst

```diff
@@ -1,161 +1,208 @@
+.. image:: third_party_shipper/shipping-connectors.png
    :align: center
+   :alt: Options of available shipping connectors in Odoo.
 
-Then click on **Apply**.
+Then, click :guilabel:`Save`.
 
 .. note::
-    If you want to integrate delivery methods in your e-commerce,
-    you'll have to install the **eCommerce Delivery** module.
+   Delivery methods can also be integrated with operations in the *eCommerce* app. (installation
+   details found :ref:` here <general/install>`).
+
+.. _inventory/shipping_receiving/configure-delivery-method:
 
-Configure the delivery method
------------------------------
+Delivery method
+---------------
 
-To configure your delivery methods, go to the **Inventory** module, click on
-:menuselection:`Configuration --> Delivery Methods`.
+Next, add the carrier details by going to :menuselection:`Inventory app --> Configuration -->
+Shipping Methods`.
 
-The delivery methods for the chosen shippers have been automatically
-created. Most of the time, there will be **2** delivery methods for the same
-provider: one for **international** shipping and the other for **domestic**
-shipping.
+.. note::
+   The list often includes **two** delivery methods from the same :guilabel:`Provider`: one for
+   international and one for domestic shipping.
 
-You can create other methods with the same provider with other
-configuration, for example the **Packaging Type**.
+   Additional delivery methods can be created for specific purposes, such as :doc:`packaging
+   <../../product_management/product_tracking/packaging>`.
 
-You can change the **Name** of the delivery method. This is the name that
-will appear on your ecommerce.
+.. seealso::
+   :doc:`Configure delivery methods <../setup_configuration/delivery_method>`
 
-Flag **Shipping enabled** when you are ready to use it.
+The delivery method must be published before it can be used. To do so, choose the desired delivery
+method.
 
-.. image:: third_party_shipper/third_party_shipper02.png
+.. image:: third_party_shipper/publish-delivery-method.png
    :align: center
+   :alt: Display list of delivery methods and the "is Published" column.
 
-In the pricing tab, the name of the provider chosen under the **Price computation**
-means that the pricing will be computed by the shipper system.
-
-The configuration of the shipper is split into two columns:
+On the shipping method page, configure the :doc:`delivery method details
+<../setup_configuration/delivery_method>`, such as :guilabel:`Integration Level`,
+:guilabel:`Delivery Product`, and :guilabel:`Invoicing Policy`.
 
--  The first one is linked to **your account** (developer key, password,...).
-   For more information, please refer to the provider website.
+Then, in the :guilabel:`Configuration` tab, input the API credentials and password with the shipping
+carrier. Then, go to the :guilabel:`Destination Availability` tab and configure the service coverage
+of the shipping carrier.
```

Comment:
For these instructions here, do you think some additional instruction on what these are, how you find them, and what they affect would be helpful? I know it's different per carrier, but maybe a .. note:: or a .. tip:: saying "read the doc for individual carriers to find this stuff" would be nice